trade-finance-manager-ui/server/helpers/premium-payments-table-checkbox-id-helper.ts

```diff
@@ -1,14 +1,17 @@
 import { CURRENCY, Currency, FEE_RECORD_STATUS, FeeRecordStatus } from '@ukef/dtfs2-common';
 import { PremiumPaymentsTableCheckboxId } from '../types/premium-payments-table-checkbox-id';
 
+const COMMA_SEPARATED_IDS_REGEX_GROUP = `(?<commaSeparatedIds>(\\d+,?)+)`;
+const COMMA_SEPARATED_IDS_REGEX = new RegExp(COMMA_SEPARATED_IDS_REGEX_GROUP);
```

Comment:
Can we add some documentation about the regex please

Reply:
Added docs to each of the regex group constants in this file (2 additional ones below this)